tests/test_level2.py

```diff
@@ -49,7 +49,7 @@ def test_SMOSL2(test_output_path):
         assert str(t.astype("datetime64[D]")) == "2010-06-01"
 
     # test repurpose
-    reader.repurpose(outpath, overwrite=True, n_proc=4,)
+    reader.repurpose(outpath, overwrite=True, n_proc=1,)
```

Comment:
This one I made only to make the test pass. Don't really understand what's happening here

Reply:
does it mean that parallel processing is not working?

Reply:
we should make sure that this works

Reply:
I don't think there is any problem with n_proc=4, can you point out a CI run where it caused an error?

Reply:
ah, you already posted that ;)

Reply:
and if that helps, change the repurpose function accordingly please

Reply:
you mean changing it in the test or there is another place in the package I should update the repurpose function?

Reply:
I mean not just setting the keyword in the test, which would only cover up the problem, but making sure that it's set in the repurpose function of the reader, that is tested